src/Controls/samples/Controls.Sample/Maui.Controls.Sample.csproj

```diff
@@ -34,6 +34,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Http" Version="7.0.0-rc.2.22472.3" />
```

Comment:
We need to add this to Versions.props

Reply:
I wouldn't recommend using Microsoft.Extensions.Http on mobile, as it hurts startup time quite a bit:

https://github.com/dotnet/runtime/issues/66863

Reply:
Sorry, I didn't read what file this was.

It is fine as long as it is optional, because this is probably not as much of an issue on Desktop.